modules/ROOT/pages/apikit-4-xml-reference.adoc

```diff
@@ -1,91 +1,97 @@
 |===
 
-=== apikit:config Attributes
+=== `apikit:config` Attributes
 
-These elements are of type NonBlankString.
+The `apikit:config` attributes are of `NonBlankString`type`.
 
 [%header,cols="22a,10a,68a"]
 |===
 |Attribute Name|Required |Description
-|*outboundHeadersMapName* |N |Used to identify the name of the map the contains response headers
-|*httpStatusVarName* |N |The name of the variable that stores the HTTP status code in the response
-|*keepRamlBaseUri* |N | An alias for the equivalent "keepApiBaseUri" attribute.
+|`outboundHeadersMapName` |N |Name of the map the contains response headers.
+|`httpStatusVarName` |N |Name of the variable that stores the HTTP status code in the response.
+|`keepRamlBaseUri` |N |Alias for the equivalent `keepApiBaseUri` attribute.
 
-This attribute is deprecated since APIkit 1.2.0 and only kept for compatibility with APIkit 1.1.x. +
-It's recommended that API implementations use "keepApiBaseUri" instead of “keepRamlBaseUri”.
-|*name* |Y |Configuration name that other elements, such as apikit:router, reference
-|*raml* |Y | An alias for the equivalent "api" attribute.
+This attribute is deprecated since APIkit 1.2.0. Included to provide compatibility with APIkit 1.1.x. +
+Use `keepApiBaseUri` instead of `keepRamlBaseUri` in your API implementations.
+|name |Y |Configuration name that other elements, such as `apikit:router`, reference.
+|raml |Y |Alias for the equivalent `api` attribute.
 
-This attribute is deprecated since APIkit 1.2.0 and only kept for compatibility with APIkit 1.1.x. +
-It's recommended that API implementations use "api" instead of “raml”.
-|*parser* | N | Specifies the parser used to validate requests.
+This attribute is deprecated since APIkit 1.2.0. Included to provide compatibility with APIkit 1.1.x. +
+Use `api` instead of `raml` in your API implementations.
+|parser | N |Parser used to validate requests.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
 
 Possible values are:
 
-* AUTO: The APIkit Router will try to initiate using AMF, it's not possible (it has errors), then it will try with the RAML Parser.
-* AMF: The APIkit Router will use AMF to read the API and validate incoming requests.
-* RAML: The APIkit Router will use RAML Parser to read the API and validate incoming requests. This is similar to previous APIkit versions.
+* AUTO
++
+*APIkit Router* initiates by using AMF. If it has errors, it initiates by using the RAML parser.
+* AMF
++
+*APIkit Router* uses AMF to read the API and validate incoming requests.
+* RAML
++
+*APIkit Router* uses RAML parser to read the API and validate incoming requests. Similar to previous APIkit versions.
 
-*Default Value: AUTO*.
+Default Value: AUTO.
 
-| *api* | N | Root API file location.
+| api | N | Root API file location.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
```

Comment:
Can you make this a sentence (add a verb)?

Reply:
OK! Thanks!